README.md

```diff
@@ -351,6 +352,42 @@ make stop
 make run
 ```
 
+### How do I tear down a cluster on a cloud provider?
+
+The [SCF secret generator](https://github.com/SUSE/scf-secret-generator)
+creates secrets in the CF and UAA namespaces, and helm doesn't know about
+these, which means they won't be deleted if the release is deleted.  The best
+way to remove everything is to run the following commands:
+
+```bash
+helm delete --purge ${CF_RELEASE_NAME}
+kubectl delete namespace ${CF_NAMESPACE}
+helm delete --purge ${UAA_RELEASE_NAME}
+kubectl delete namespace ${UAA_NAMESPACE}
+```
+
+However, busy systems may encounter timeouts when the release is deleted:
+
+```bash
+$ helm delete --purge scf
+E0622 02:27:17.555417   14014 portforward.go:178] lost connection to pod
+Error: transport is closing
+```
+
+In this case, deleting the StatefulSets before anything else will make the
+operation more likely to succeed:
+
+```bash
+kubectl delete statefulsets --all --namespace ${CF_NAMESPACE}
+helm delete --purge ${CF_RELEASE_NAME}
+kubectl delete namespace ${CF_NAMESPACE}
+kubectl delete statefulsets --all --namespace ${UAA_NAMESPACE}
+helm delete --purge ${UAA_RELEASE_NAME}
+kubectl delete namespace ${UAA_NAMESPACE}
+```
```

Comment:
Is this something we should put into our `make/stop` scripts ?

Reply:
I don't think so... I've been testing this yesterday and I haven't been able to get the Vagrant box to break, even under high load. And I think this will probably get fixed by helm / kube in the future, so I'd rather not have the workaround running all the time if we don't need it.